experimental/dds/tree2/src/feature-libraries/sequence-field/sequenceFieldToDelta.ts

```diff
@@ -74,9 +94,24 @@ function cellDeltaFromMark<TNodeChange>(
 				return mark.count;
 			}
 			case "Delete": {
+				const detachedField = repairDataBuilder.handler({
+					revision: mark.revision,
+					localId: mark.id,
+				});
+				const moveId = brandOpaque<Delta.MoveId>(idAllocator());
+				repairDataBuilder.marks.set(detachedField, [
+					{
+						type: Delta.MarkType.MoveIn,
+						count: mark.count,
```

Comment:
Unfortunately, we need to create `mark.count` move-out/move-in pairs of count 1 instead of creating a single pair of count `mark.count`.
This is because addressing the repair data will not work well otherwise: if the nth node is revived then all the nodes after that would end up at a shifted index.
We'll add support for batching later, but for now a loop will do.